csv_detective/detect_fields/FR/geo/code_fantoir/__init__.py

```diff
@@ -1,15 +1,7 @@
-from os.path import dirname, join
-import re
+from frformat import CodeFantoir
 
 PROPORTION = 1
-f = open(join(dirname(__file__), 'code_fantoir.txt'), 'r')
-codes_fantoir = f.read().split('\n')
-f.close()
 
-
-def _is(val):
-    '''Renvoie True si val peut être un code FANTOIR/RIVOLI, False sinon'''
-    if not bool(re.match(r'^[0-9A-Z][0-9]{3}[ABCDEFGHJKLMNPRSTUVWXYZ]$', val)):
-        return False
-
-    return val[:4] in codes_fantoir
+def _is (val):
+    instance = CodeFantoir()
+    return instance.is_valid(val)
```

Comment:
`is_valid` is a classmethod and don't need an instanciation

Reply:
Done